src/Button.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import "./Button.css";
+
+function Button(props) {
+  return <div className="button">{props.children}</div>;
```

Comment:
this should be a `button` element, because you rely on its functionality and semantic value